src/io/io.cpp

```diff
@@ -64,11 +75,9 @@ void Create_Log_File(struct parameters P)
 
 void Write_Message_To_Log_File(const char *message)
 {
-#ifdef MPI_CHOLLA
-  if (procID != 0) {
+  if (!!Is_Root_Proc()) {
```

Comment:
Is the double negation intended? 

Also, several months ago there was a discussion on using keyword vs. character Boolean operators (! vs not, || vs or, etc). If I recall correctly we decided to use keyword operators.

Reply:
The double negative was definitely an oversight. Thanks for catching that.

Gotcha, I'll shift to using `not` and `or`